contracts/lib/Common.sol

```diff
@@ -32,7 +32,16 @@ library Common {
   }
 
   // convert bytes to uint8
-  function toUint8(bytes _arg) public view returns (uint8) {
+  function toUint8(bytes _arg) internal view returns (uint8) {
     return uint8(_arg[0]);
   }
+
+  /**
+  * @dev Returns the maximum of two numbers
+  * @param a uint256 number
+  * @param b uint256 number
+  */
+  function max(uint256 a, uint256 b) internal pure returns (uint256) {
```

Comment:
why not use library?
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/math/Math.sol#L9

Reply:
Yeah I will add. Missed it.

Reply:
Added in new commit https://github.com/maticnetwork/contracts/pull/16/commits/9d65877b2d7c1e7744244c9b01a3288291694f70